src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -418,6 +417,12 @@ private function addSubForm(FormBuilderInterface $builder, $name, ChoiceView $ch
             'block_name' => 'entry',
         );
 
+        // Resolve label_attr to the right level
+        if (isset($choiceOpts['attr']['label_attr'])) {
+            $choiceOpts['label_attr'] = $choiceOpts['attr']['label_attr'];
+            unset($choiceOpts['attr']['label_attr']);
```

Comment:
this is not the correct thing. the `attr` is for html attributes. but `label_attr` doesn't exist as html attr.
To support your use-case, we would need to add another option: `choice_label_attr` with the same logic as the other choice options.